### ISSUE TITLE
SYNCED (1008080)

### ISSUE THREAD

**Issue by gabriele2000** [ValveSoftware/Proton#7084]
# Compatibility Report
- Name of the game with compatibility issues: SYNCED
- Steam AppID of the game: 1008080

## System Information
- GPU:  GTX1050TI
- Video driver version:  535.104.05
- Kernel version:  6.4.11-76060411-generic
- Link to full system information report as [Gist](https://gist.github.com/): [system spec - gist](https://gist.github.com/gabriele2000/85f19e47d767078d12bc4a7322cc3450)
- Proton version: GE-Proton-8.14

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-1008080.zip](https://github.com/ValveSoftware/Proton/files/12566599/steam-1008080.zip)

## Symptoms 
Launcher launches, upon clicking "play", the launcher will close and the process will terminate.
TLDR - Game doesn't open at all

## Reproduction
1 - Click "play"
2 - Launcher will appear
3 - Click "launch"
4 - Lose hope waiting for the game that will never open

**Comment by pollux78** [comment]
my launcher doesnt even appear, just sits in the background and does nothing

**Comment by gabriele2000** [comment]
> my launcher doesnt even appear, just sits in the background and does nothing

you need to terminate the process, wait a few seconds then re-open it

**Comment by pollux78** [comment]
> > my launcher doesnt even appear, just sits in the background and does nothing
> 
> you need to terminate the process, wait a few seconds then re-open it

I ended up getting the launcher to show up but after clicking launch it doesn't go any further sadly

**Comment by Megalomaniak** [comment]
>  that I haven't found an existing compatibility report for this game.

There are reports on ProtonDB by now, but only successful runtime claims are from SteamDeck. Doesn't seem to run successfully on anything but the SteamDeck.

I launched with `PROTON_LOG=1 %command%` and did get a log out of it:
https://gist.github.com/Megalomaniak/65550a2ba0419ad9e370d9b34fecd5d6

But this 309k+ lines monstrosity is beyond me.
